src/Formatter/IntlMoneyFormatter.php

```diff
@@ -16,13 +16,19 @@
      * @var \NumberFormatter
      */
     private $formatter;
+    /**
+     * @var int|null
```

Comment:
It is good to have available, but why should it be required? There was nothing wrong with the old behaviour. Only it does not match everyone's needs. This fixes that.